pkg/kv/kvprober/kvprober.go

```diff
@@ -20,6 +20,7 @@ package kvprober
 import (
 	"context"
 	"math/rand"
+	"regexp"
```

Comment:
Is your plan to merge this PR with no full trace dumping in case of errors & then add that in a follow up PR, if time permits? That seems like a good plan. Just confirming.

Reply:
Yes, that's the plan. Ideally should be a small addition.